.gitignore

```diff
@@ -2,9 +2,5 @@
 *networkTest.scd
 # ignore all pyc files
 *.pyc
-# ignore everything inside tests directory
```

Comment:
I think _networkArgs.json_ and _modalData.json_ should be added here. Otherwise they will appear as untracked files to _Git_.

Reply:
I think it could be better to use Platform.defaultTempDir if the files aren't needed later or the path is not set to an specific directory, or use home as default. Because the user is writing data to the code directory and also is a bit hidden there.

Reply:
Right, that would definitely work for _networkArgs.json_. The problem is that _modalData.json_ I do want to keep sometimes. Especially for large networks it can take a while to do the computation (minutes). So then it is kind of nice to check if the file already exists and if so, not having to run the offline computation part again. But writing data to the code directory isn't an ideal solution for sure. Even more because of the fact that _modalData.json_ is not uniquely identifiable for instance. If I generate it for a particular network and want to cache it, but then do the computation for another network it will overwrite _modalData.json_ again. So the whole way _modalData.json_ is handled should probably be rethought to be honest, but that is going to be more work of course.

So in order to not break any existing / expected behaviour I would prefer to just add these 2 files to _.gitignore_, even though I agree this could be handled better.

Reply:
Very sorry for the late reply, I forgot about it. If you want you can resolve this two issues it's ok by me and can't commit more changes.